### PR TITLE
WinUI-3213 : Updated seting the value content in UG document content for SfMaskedTextBox control

### PR DESCRIPTION
**Pull request by Vignesh-SF3580**
### Description
Updated setting the value in the getting started.
Updated setting the prompt character in getting started.

### Solution Description
Added setting the value content.
Added setting the prompt character content.
Added the output images for the value and the prompt char.

### CODE REVIEW

File: winui/Masked-TextBox/Getting-Started.md [hunk 1]
```diff
@@ -90,4 +90,49 @@ maskedTextBox.Mask = "[A-Za-z0-9._%-]+@[A-Za-z0-9]+\\.[A-Za-z]{2,3}";
 {% endhighlight %}
 {% endtabs %}
 
-![RegEx mask in WinUI MaskedTextBox](MaskedTextBox_Images/winui_regex_mask.png)
\ No newline at end of file
+![RegEx mask in WinUI MaskedTextBox](MaskedTextBox_Images/winui_regex_mask.png)
+
+## Setting the prompt character
+
+Displays prompt character for the absence of your input in the mask and its default value is ‘_’. You can set the custom prompt character using `PromptChar` property.
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Getting-Started.md:97]: @Vignesh-SF3580 Every single mask displays a prompt character ('_') in the absence of your input. You can customize this prompt character by using the `PromptChar` property."

File: winui/Masked-TextBox/Getting-Started.md [hunk 2]
```diff
@@ -90,4 +90,49 @@ maskedTextBox.Mask = "[A-Za-z0-9._%-]+@[A-Za-z0-9]+\\.[A-Za-z]{2,3}";
...
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI MaskedTextBox prompt character](MaskedTextBox_Images/winui_masked_textbox_prompt_char.png)
+
+## Setting the value
+
+The MaskedTextBox control displays the value that can be set using the `Value` property.
+
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Getting-Started.md:117]: @Vignesh-SF3580 The `Value` property is used to set the input value for MaskedTextBox control.

File: winui/Masked-TextBox/Getting-Started.md (hunk 1 above)
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Getting-Started.md]: Changed the content for setting the prompt character.

File: winui/Masked-TextBox/Getting-Started.md (hunk 2 above)
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Getting-Started.md:117]: Changed the content for setting the value.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**